### PR TITLE
create crud

### PR DESCRIPTION
**Pull request by serjmain**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const router = require("./routes/routes");
+const crimeRepository = require("./models/crime");
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const swaggerUI = require("swagger-ui-express");
+const swaggerJsDoc = require("swagger-jsdoc");
+const client = require("./service/dbservice");
+
+const options = {
```
Comment (korolovdmitriy) [inline app.js:10]: options to config,json

File: app.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+app.listen(3000, async () => {
+  await client.connect();
+  crimeRepository.init();
+  console.log(`Server is running on http://localhost:${3000}`);
+})
+
+app.use(express.json());
+app.use(express.urlencoded({extended: true}));
+app.use("/", router);
```
Comment (korolovdmitriy) [inline app.js:39]: Need a middleware function to handle errors

**Review by korolovdmitriy — APPROVED** [review]

**Review by Liza-14 — APPROVED** [review]

File: app.js (hunk 1 above)
Comment (serjmain) [reply app.js:10]: Created swagger-config.json n replaced options

File: app.js (hunk 2 above)
Comment (serjmain) [reply app.js:39]: already done

**Pull request closed without merging**